src/content/docs/usage/runners.mdx

```diff
@@ -18,19 +18,19 @@ Daytona provides an option to register a new Runner, allowing you to deploy it o
 1. Run the following command to register a new Runner:
 
 ```shell
-daytona runner register
+daytona server runner register
 ```
 
 Upon running the command, Daytona will generate a code snippet containing the server API URL and API key required to deploy the Runner on the target system.
 
-2. Paste the code snippet into the target system's terminal and execute it to deploy the Runner.
+2. Paste the code snippet into the target system's terminal and execute it to configure the Runner.
 
 ## Start a Runner
 
-Once the Runner is deployed on the target system, you can start it by running the following command:
+Once the Runner is configured on the target system, you can start it by running the following command:
 
 ```shell
-daytona runner start
+daytona server runner start
```

Comment:
It's actually `daytona runner start`.

It might be a bit confusing 😅 

`server runner` commands are those managed by the server and `runner` commands are those that are related to the runner on that machine.

`daytona runner start` starts the runner that is installed on the machine while `daytona server runner register` tells the server to register a new runner.